Dockerfile

```diff
@@ -68,6 +68,16 @@ WORKDIR /home/builder
 USER builder
 ENV PACKAGE=${PACKAGE} ARCH=${ARCH}
 COPY --chown=builder roles/${REPO}.root.json ./rpmbuild/BUILD/root.json
+# We attempt to copy `Licenses.toml` and `licenses` for the current build, otherwise
+# an empty file and a directory are created. Packages that use `bottlerocket-license-tool`
+# will fail the build, since we expect users to provide their own license
+RUN --mount=target=/host \
+  ( [ -f /host/Licenses.toml ] \
+  && cp /host/Licenses.toml ./rpmbuild/BUILD/ \
+  || touch ./rpmbuild/BUILD/Licenses.toml ) \
```

Comment:
Is this necessary given that you tested for the existence of /host/Licenses.toml before copying it to here? (Same goes for the directory /host/licenses)

```suggestion
  || touch ./rpmbuild/BUILD/Licenses.toml ) \
```